data/migrations/0079_purchase_family.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 4.0.5 on 2022-06-14 15:29
+
+from django.db import migrations, models
+
+
+def migrate_product_family(apps, schema_editor):
+    Purchase = apps.get_model("data", "Purchase")
+    mapping = {
+        "VIANDES_VOLAILLES": "VIANDES_VOLAILLES",
+        "PRODUITS_DE_LA_MER": "PRODUITS_DE_LA_MER",
+        "FRUITS_ET_LEGUMES": "AUTRES",
+        "PRODUITS_CEREALIERS": "",
+        "ENTREES": "AUTRES",
+        "PRODUITS_LAITIERS": "PRODUITS_LAITIERS",
+        "BOISSONS": "BOISSONS",
+        "AIDES": "AUTRES",
+        "BEURRE_OEUF_FROMAGE": "PRODUITS_LAITIERS",
+        "PRODUITS_SUCRES": "",
+        "ALIMENTS_INFANTILES": "AUTRES",
+        "GLACES_SORBETS": "AUTRES",
+        "AUTRES": "AUTRES",
+    }
+    for purchase in Purchase.objects.all():
+        if purchase.category and mapping[purchase.category]:
+            purchase.family = mapping[purchase.category]
+            purchase.save()
+
+
+def undo_migrate(apps, schema_editor):
```

Comment:
Nano-détail, par pseudo-convention sur les migrations on prend le même nom de la fonction avec `undo_` avant

```suggestion
def undo_migrate_product_family(apps, schema_editor):
```